models/src/it/java/io/sphere/sdk/shippingmethods/queries/GetShippingMethodsByCartTest.java

```diff
@@ -0,0 +1,21 @@
+package io.sphere.sdk.shippingmethods.queries;
+
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.shippingmethods.ShippingMethod;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import java.util.List;
+
+import static io.sphere.sdk.carts.CartFixtures.*;
+
+
+public class GetShippingMethodsByCartTest extends IntegrationTest {
+    @Test
+    public void execution() throws Exception {
+        final Cart cart = createCartWithShippingAddress(client());
+
+        final List<ShippingMethod> result =
+                execute(GetShippingMethodsByCart.of(cart));
```

Comment:
No assertions to check that the returned shipping methods, or at least the size of the list, is correct?